lib/eq3interface.js

```diff
@@ -11,60 +11,301 @@ const status = {
   lowBattery: 128,
 }
 
+// convert any number to 2 digits hex.
+// ensures integer, and takes last two digits of hex conversion with zero filling to 2 if < 16
+var h2 = function(val) {
```

Comment:
Is broken
```suggestion
var h2 = function(number) {
```

Reply:
hah, never trust an englishman!  Had a suspicion that something was up with my rads; now I know - the time has probably been s****d for a month!  (actually, date/time seems correct; but I've not run anything more complex than polling since the changes....)